core/proto.go

```diff
@@ -54,6 +54,12 @@ func ParSignedDataFromProto(typ DutyType, data *pbv1.ParSignedData) (ParSignedDa
 			return ParSignedData{}, errors.Wrap(err, "unmarshal block")
 		}
 		signedData = b
```

Comment:
i looked for this ParSignedDataFromProto function in the repo and this is it's only declaration so not sure if this needs to be implemented here

Reply:
`ParSignedDataFromProto ` is only called from `ParSignedData<Set>FromProto` which is called from parsigex to unmarshal protos received over the wire. But yeah, adding it here is correct.